lib/index.d.ts

```diff
@@ -1,252 +1,257 @@
 // TypeScript Version: 2.1
 
-import Headers from 'fetch-ponyfill';
+/** stardog.js: The Stardog JS API*/
 
-declare global {
-  
-  /** stardog.js: The Stardog JS API*/
-  namespace Stardog {
+import Headers from 'fetch-ponyfill';
 
-    type ContentMimeTypes =
+namespace HTTP {
+    export type ContentMimeTypes =
         'application/x-turtle' |
         'text/turtle' |
         'application/rdf+xml' |
         'text/plain' |
         'application/x-trig' |
         'text/x-nquads' |
         'application/trix';
-        
-    type AcceptMimeTypes =
+
+    export type AcceptMimeTypes =
         'text/plain' |
         'application/json' |
         'text/boolean';
 
-    interface HTTPMessage {
+    export interface Message {
         status: string;
         statusText: string;
     }
 
-    interface HTTPBody {
+    export interface Body {
         status: string;
         statusText: string;
-        result: Object | string | boolean | null;
-    }
-    
-    interface ConnectionOptions {
-        endpoint: string;
-        username: string;
-        password: string;
+        result: object | string | boolean | null;
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html